firestore/integration_test_internal/src/aggregate_query_snapshot_test.cc

```diff
@@ -46,90 +46,339 @@ std::size_t AggregateQuerySnapshotHash(const AggregateQuerySnapshot& snapshot) {
 
 namespace {
 
-TEST_F(AggregateQuerySnapshotTest, DefaultConstructor) {
+TEST_F(AggregateQuerySnapshotTest, DefaultConstructorReturnsValidObject) {
   AggregateQuerySnapshot snapshot;
   EXPECT_EQ(snapshot.query(), AggregateQuery());
   EXPECT_EQ(snapshot.count(), 0);
   EXPECT_FALSE(snapshot.is_valid());
 }
 
-TEST_F(AggregateQuerySnapshotTest, CopyConstructor) {
-  const Query& query = TestFirestore()->Collection("foo").Limit(10);
-  const AggregateQuery& aggregate_query = query.Count();
+TEST_F(AggregateQuerySnapshotTest,
+       CopyConstructorAppliedToDefaultObjectReturnsEqualObject) {
+  AggregateQuerySnapshot snapshot = AggregateQuerySnapshot();
```

Comment:
In C++, to invoke the default constructor during value initialization you can just omit `= AggregateQuerySnapshot()` and make the line more succinctly `AggregateQuerySnapshot snapshot;`.